pkg/redact/literal.go

```
@@ -34,7 +39,8 @@ func (r literalRedactor) Redact(input io.Reader, path string) io.Reader {
 			if err == nil || err == io.EOF {
 				writer.Close()
 			} else {
-				if err == bufio.ErrTooLong {
+				// Check if error is about line exceeding maximum size
+				if err != nil && bytes.Contains([]byte(err.Error()), []byte("exceeds maximum size")) {
```

can we return a typed error to avoid having to parse this, or even return bufio.ErrTooLong directly?